customersupport/customersupport_tests.py

```diff
@@ -47,41 +48,31 @@ def test_sales_orderinfo(self):
 
     def test_sales_orders_called(self):
         """ tests for Sales Search Order API is called """
-        try:
-            r = requests.get(SALES_URL+'/order/search')
-            r.raise_for_status()
-            assert r.status_code == 200
-        except ConnectionError:
-            r.raise_for_status()
+        actual = sales.get_order_info("John doe 1111 street Rochester NY 14568", True)
```

Comment:
Because this method has multiple parameters, the `True` here will actually go to the `incl_payment_info` parameter. 
Please be explicit with the keyword arguments. To use the mocked version, this should be:
`actual = sales.get_order_info("John doe 1111 street Rochester NY 14568", mock=True)`

Change this and all of the others that do the same thing.